hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestOmDBInsightEndPoint.java

```diff
@@ -693,4 +694,58 @@ public void testGetDeletedDirInfo() throws Exception {
     Assertions.assertEquals("/sampleVol/bucketOne/dir_two",
         keyInsightInfoResp.getLastKey());
   }
+
+  @Test
+  public void testGetDirectorySizeInfo() throws Exception {
+
+    OmKeyInfo omKeyInfo1 =
+        getOmKeyInfo("sampleVol", "bucketOne", "dir1", false);
+    OmKeyInfo omKeyInfo2 =
+        getOmKeyInfo("sampleVol", "bucketTwo", "dir2", false);
+    OmKeyInfo omKeyInfo3 =
+        getOmKeyInfo("sampleVol", "bucketThree", "dir3", false);
+
+    // Add 3 entries to deleted dir table for directory dir1, dir2 and dir3
+    // having object id 1, 2 and 3 respectively
+    reconOMMetadataManager.getDeletedDirTable()
+        .put("/18/28/22/dir1/1", omKeyInfo1);
```

Comment:
@ArafatKhan2198  thanks for working on this patch. Pls update minor change to correct parent Id same as bucket Id based on test data we are inserting.

Reply:
Done!